great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -344,9 +347,9 @@ def get_batch_data_and_markers(  # noqa: C901 - 22
             reader_method = batch_spec.reader_method
             reader_options = batch_spec.reader_options
             reader_fn = self._get_reader_fn(reader_method)
-            reader_fn_result: pd.DataFrame | list[pd.DataFrame] = reader_fn(
-                **reader_options
-            )
+            reader_fn_result: pd.DataFrame | list[
+                pd.DataFrame
+            ] = execute_pandas_reader_fn(reader_fn, reader_options)
```

Comment:
Interested in how this works out / whether it works with the `pandas_sql` case

Reply:
Based on local testing it does work to ignore the warnings.